evaluate/metrics/assessment.go

```diff
@@ -16,19 +16,19 @@ var (
 	// AllAssessmentKeysStrings returns all registered assessment keys as strings.
 	AllAssessmentKeysStrings []string
 
-	// pointsPerAssessment holds the points awarded for a specific assessment.
```

Comment:
COmmit subject "Redo the coverage logic" same rules as internally. What, and why.